src/components/views/messages/HiddenBody.tsx

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright 2020 - 2021 The Matrix.org Foundation C.I.C.
```

Comment:
```suggestion
Copyright 2022 The Matrix.org Foundation C.I.C.
```

Reply:
That's what happens when I take too long to push a PR for review :)